crates/chain/src/indexed_tx_graph.rs

```diff
@@ -181,64 +181,64 @@ where
     )
 )]
 #[must_use]
-pub struct IndexedAdditions<A, IA> {
-    /// [`TxGraph`] additions.
-    pub graph_additions: Additions<A>,
-    /// [`Indexer`] additions.
-    pub index_additions: IA,
+pub struct ChangeSet<A, IA> {
+    /// [`TxGraph`] changeset.
+    pub graph_changeset: tx_graph::ChangeSet<A>,
+    /// [`Indexer`] changeset.
+    pub index_changeset: IA,
```

Comment:
It's inside a thing that's a changeset so I'd rather remove this suffix.

```suggestion
    pub graph: tx_graph::ChangeSet<A>,
    /// [`Indexer`] changeset.
    pub indexer: IA,
```